packages/node_modules/@webex/webex-core/test/unit/spec/services/service-catalogs.js

```diff
@@ -190,45 +190,154 @@ describe('webex-core', () => {
           })
         );
 
-        hostFixtureA = fixture.hostCatalog['example-service-a'][0];
-        hostFixtureB = fixture.hostCatalog['example-service-b'][1];
+        remoteHosts = fixture.hostCatalog['example-service-c'];
+        localHosts = [
+          ...fixture.hostCatalog['example-service-a'],
+          ...fixture.hostCatalog['example-service-b']
+        ];
       });
 
       it('should return all hosts when called without params', () => {
-        filteredHosts = serviceCatalogs.filterCluster();
+        filteredHosts = serviceCatalogs.filterLocal();
 
         assert.equal(filteredHosts.length, serviceCatalogs.hosts.length);
         assert.deepEqual(filteredHosts, serviceCatalogs.hosts);
       });
 
-      it('should return only service hosts in the specific cluster', () => {
-        filteredHosts = serviceCatalogs.filterCluster(hostFixtureA.id);
+      it('should return only local hosts when called with true', () => {
+        filteredHosts = serviceCatalogs.filterLocal(true);
 
-        assert.equal(filteredHosts.length, 1);
-        assert.equal(filteredHosts[0].id, hostFixtureA.id);
+        assert.equal(filteredHosts.length, localHosts.length);
+        assert.isTrue(filteredHosts.every(
+          (host) => host.local === true
+        ));
+      });
+
+      it('should return only hosts remote hosts when called with false', () => {
+        filteredHosts = serviceCatalogs.filterLocal(false);
+
+        assert.equal(filteredHosts.length, remoteHosts.length);
+        assert.isTrue(filteredHosts.every(
+          (host) => host.local === false
+        ));
+      });
+    });
+
+    describe('#filterPriority()', () => {
+      let filteredHosts;
+      let priorityHosts;
+
+      beforeEach('generate the service host class objects', () => {
+        serviceCatalogs.load(
+          ServiceCatalogs.mapRemoteCatalog({
+            catalog: SERVICE_CATALOGS[0],
+            ...fixture
+          })
+        );
+
+        priorityHosts = [
+          fixture.hostCatalog['example-service-a'][0],
+          fixture.hostCatalog['example-service-b'][0],
+          fixture.hostCatalog['example-service-c'][0]
+        ];
+      });
+
+      it('should return all hosts when called without params', () => {
+        filteredHosts = serviceCatalogs.filterPriority();
+
+        assert.equal(filteredHosts.length, serviceCatalogs.hosts.length);
```

Comment:
This test is unnecessary with the following `deepEqual`.

Reply:
Addressing this, I see how it is redundant.